tests/util/test_file.cpp

```diff
@@ -51,10 +51,14 @@ TestFile::TestFile()
         const char* dir = getenv("TMPDIR");
         if (dir && *dir)
             return dir;
+#ifdef REALM_ANDROID
```

Comment:
`REALM_ANDROID` is always defined. It should be checked using `#if REALM_ANDROID` rather than `#ifdef REALM_ANDROID`.